luminoso_api/client.py

```diff
@@ -395,13 +379,13 @@ def change_path(self, path):
             url = self.root_url + path
         else:
             url = self.url + path
-        return self.__class__(self._auth, url)
+        return self.__class__(self.session.auth, url)
```

Comment:
Hm.  I really don't like this change_path thing that makes a new client object anyway.  But yeah, I think I could make it take a session instead of an auth object, now that the session and the auth object don't have circular references or whatever they had before.